clients/drcachesim/tracer/instru_online.cpp

```diff
@@ -178,13 +178,13 @@ online_instru_t::append_thread_header(byte *buf_ptr, thread_id_t tid,
     new_buf += append_marker(new_buf, TRACE_MARKER_TYPE_FILETYPE, file_type);
     new_buf += append_marker(new_buf, TRACE_MARKER_TYPE_CACHE_LINE_SIZE,
                              proc_get_cache_line_size());
+    new_buf += append_marker(new_buf, TRACE_MARKER_TYPE_PAGE_SIZE, dr_page_size());
 #if defined(AARCH64)
```

Comment:
Since this is still inside a function named append_thread_header, maybe add a comment highlighting that there may be other instances too in the trace besides the one in the header, and that this marker should preferably be the last thing emitted by the header (assuming that's what we prefer).

Reply:
I have added a comment that there might be later instances of the marker.